README.md

```diff
@@ -19,13 +52,42 @@ The project documentation is available at https://pages.nist.gov/dioptra/.
 - [License](#license)
 <!-- markdownlint-enable MD007 MD030 -->
 
-## User setup
+### User setup
+
+If you are have been provided with an existing Dioptra deployment, skip ahead to the [User quickstart](#user-quickstart) section.
+
+#### Build the containers
+
+The first step in setting up Dioptra is to clone the repository and build the docker containers for the various services that are part of a deployment.
+See the [Building the containers](https://pages.nist.gov/dioptra/getting-started/building-the-containers.html) section of the documentation for instructions.
+
+#### Run Dioptra
+
+Once the containers have been built, the next step is to configure the deployment with cruft and run Dioptra.
+See the [Running Dioptra](https://pages.nist.gov/dioptra/getting-started/running-dioptra.html) section of the documentation for instructions
+
+Additionally, you may want to add datasets and configure some of the provided examples in your deployment. See the [Downloading datasets](https://pages.nist.gov/dioptra/getting-started/acquiring-datasets.html) and [Examples](https://pages.nist.gov/dioptra/getting-started/examples-setup.html) sections of the documentation.
+
+
+### User quickstart
+
+
+Read through the user guide starting with [the basics](https://pages.nist.gov/dioptra/user-guide/the-basics.html) to become familiar with the core concepts of Dioptra.
+
+#### User registration
+
+Register a user account with your Dioptra instance via either the user interface.
+The user interface can be accessed via your web browser at the URL specified when creating the deployment.
+The default is http://localhost
+
+#### Examples
+
+Consider exploring the examples available in the [examples directory](https://github.com/usnistgov/dioptra/tree/main/examples)
```

Comment:
I think you can do relative paths in the links so that you're not hardcoding them to a particular branch.

```suggestion
Consider exploring the examples available in the [examples directory](examples/)
```